test/MVCFramework45.FunctionalTests/Controllers/ManageController.cs

```diff
@@ -2,35 +2,45 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using System.Security.Claims;
-using System.Security.Principal;
-using Microsoft.AspNet.Authorization;
-using Microsoft.AspNet.Identity;
-using Microsoft.AspNet.Mvc;
-using Mvc6Framework45.FunctionalTests;
-using Mvc6Framework45.FunctionalTests.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using MVCFramework45.FunctionalTests.Models;
+using MVCFramework45.FunctionalTests.Models.ManageViewModels;
+using MVCFramework45.FunctionalTests.Services;
 
-namespace Mvc6Framework45.FunctionalTests.Controllers
+namespace MVCFramework45.FunctionalTests.Controllers
 {
     [Authorize]
     public class ManageController : Controller
     {
-        public ManageController(UserManager<ApplicationUser> userManager, SignInManager<ApplicationUser> signInManager)
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        private readonly IEmailSender _emailSender;
+        private readonly ISmsSender _smsSender;
+        private readonly ILogger _logger;
+
+        public ManageController(
+        UserManager<ApplicationUser> userManager,
+        SignInManager<ApplicationUser> signInManager,
+        IEmailSender emailSender,
+        ISmsSender smsSender,
+        ILoggerFactory loggerFactory)
         {
-            UserManager = userManager;
-            SignInManager = signInManager;
+            _userManager = userManager;
```

Comment:
Thats a good point. I did't pay much attention to these files as they were autogenerated. We should spend couple of hours to run stylecop and fix all these issues. I will create a task for it.